erpnext/templates/emails/recurring_document_failed.html

```diff
@@ -2,11 +2,10 @@
 
 <p>An error occured while creating recurring {{ type }} <b>{{ name }}</b> for <b>{{ party }}</b>.</p>
```

Comment:
just a question, don't you think it should be possible to translate this message?

Reply:
Thanks for your review, will fix it!

Reply:
sorry again but I think all sentences in the recurring_document_failed.html should have possibility to be translated

Reply:
Can you please send PR after merging of this PR?

Reply:
ok

Reply:
ok